Sources/StreamVideoSwiftUI/CallView/CornerDragableView.swift

```diff
@@ -5,22 +5,24 @@
 import StreamVideo
 import SwiftUI
 
-public struct CornerDragableView<Content: View>: View {
+public struct CornerDraggableView<Content: View>: View {
     @State var callViewPlacement = CallViewPlacement.topTrailing
-    
+
     @State private var dragAmount = CGSize.zero
-    
+
     private var scaleFactorX: CGFloat
     private var scaleFactorY: CGFloat
-    
-    var content: Content
+    private var availableFrame: CGRect
+    private var padding: UIEdgeInsets = .init(top: 8, left: 8, bottom: 8, right: 8)
```

Comment:
constant for the 8 maybe?

Reply:
TBH the padding property is already a constant as nothing is changing it and it's not available outside. Having an additional property feels too much. I'm just on the verge and happy to make the change if you think we should.

Reply:
no, no need to do that